### ISSUE TITLE
Possible to swipe the push action fully across the screen

### ISSUE THREAD

**Issue by nishtwo** [DavdRoman/DRCellSlideGestureRecognizer#18]
Hi, i have noticed that the left pull action swipe swipes all the way across the screen and the right one does. is there any way to make this consistent?

**Comment by DavdRoman** [comment]
The demo app shows multiple variations of how swipes can be set up. You can determine which `behavior` each action has (either `pull` to its original place or `push` all the way to the edge of the screen):

```obj-c
action.behavior = DRCellSlideActionPullBehavior;

// or

action.behavior = DRCellSlideActionPushBehavior;
```

**Comment by nishtwo** [comment]
yeah you you can pull to its original place or push all the way to the edge of screen but can you push to its original place or pull all the way to the edge of the screen. Eg. I want to have two actions that occur, one where you pull all the way to edge of the screen and the action fades out, and one where you push all the way to the edge of the screen and the action fades out.
